src/proj.c

```diff
@@ -1,29 +1,42 @@
 #include "proj.h"
 
 
-void project(OSQPWorkspace *work, c_float *z) {
-  c_int i, m;
+void project(OSQPWorkspace *work, OSQPVectorf *z) {
 
-  m = work->data->m;
+  OSQPVectorf_ew_bound_vec(z,z,work->data->l,work->data->u);
```

Comment:
The fact that these projection functions have essentially collapsed into 1 line that is just calling another function makes me want to suggest we just inline them from the header... but apparently C89 doesn't technically have the inline specifier (but many compilers have added it anyway)...

Reply:
Compiling with LTO perhaps makes this a moot point in any case.

Note that other implementations of this library may require more than one line, e.g. if the vector operations are partitioned across different memory segments or cores.    We are working on such a project now.